Makefile

```diff
@@ -575,10 +582,18 @@ gomodtidy:
 	@go mod tidy
 
 update-dependencies:
-	@printf $(COLOR) "Update dependencies..."
+	@printf $(COLOR) "Update dependencies (minor versions only) ..."
 	@go get -u -t $(PINNED_DEPENDENCIES) ./...
 	@go mod tidy
 
+update-dependencies-major: $(GOMAJOR)
+	@printf $(COLOR) "Major version upgrades available:"
+	@$(GOMAJOR) list -major
+	@echo ""
+	@printf $(COLOR) "Update dependencies (major versions only) ..."
+	@$(GOMAJOR) get -major all
```

Comment:
Wow. Didn't know about this tool.

Reply:
what does it mean? For example, sdk-go 1.33.1, the first 1 is major, the 33 is minor. Will it not upgrade from 1.32 to 1.33 because it is a minor version upgrade?

Reply:
Yes, this particular Target is for "major" versions _only_. The previously existing `update-dependencies` will still update "minor" versions _only_. They are orthogonal.